seed/challenges/00-getting-started/getting-started.json

```diff
@@ -645,6 +777,15 @@
           "Vérifiez votre code portefeuille. Cliquez sur votre photo dans votre coin supérieur droit. Pour activer votre code portefeuille, vous'll nécessité de lier votre compte avec code sans GitHub Camp. Votre code portefeuille affiche votre progression et combien de bons points vous avez. Vous pouvez obtenir de bons points par remplir les défis et en aidant d'autres campeurs dans nos salles de chat. Si vous obtenez Brownie Points sur plusieurs jours d'affilée, vous'll obtenez une rayure.",
           ""
         ]
+      ],
+      "TitleIt": "Configura il tuo portfolio Programmi",
+      "DescriptionIt": [
+         [
+           "//i.imgur.com/tP2ccTE.gif",
+           "Una gif che mostra come cliccare l'immagine del profilo in alto a destra per collegare a GitHub il tuo portfolio Programmi.",
+           "Controlla il tuo portfolio programmi. Clicca l'immagine in alto a destra. Per attivare il portfolio programmi, è necessario collegare il tuo account GitHub con freeCodeCamp. Il tuo portfolio programmi mostra i tuoi progressi e quanti Brownie Points hai. È possibile ottenere i Brownie Points completando i problemi e aiutando altri campers nelle nostre chat rooms. Se si ricevono Brownie Points per diversi giorni di fila, si otterrà uno \"Streak\". ",
```

Comment:
I don't really like the "portfolio programmi" translation of "code portfolio". I think one of "portfolio codice" or "portfolio progetti" would be better. Even "portfolio" alone makes much more sense to me. In any case, I think the capital "P" in the first "portfolio Programmi" is useless.